opentracing-spring-cloud-starter-zipkin/src/test/java/io/opentracing/contrib/spring/cloud/starter/zipkin/integration/ZipkinIntegrationTest.java

```diff
@@ -58,7 +58,7 @@ public void testZipkinCollectsTraces() {
   }
 
   private void assertSpanIsCorrect() {
-    await().atMost(10, TimeUnit.SECONDS).until(() -> zipkin.httpRequestCount() == 1);
```

Comment:
There could be a request to the collector which didn't store any spans

Reply:
hope it helps, I will try the release again

Reply:
Cool